ml4floods/preprocess/utils.py

```diff
@@ -25,7 +25,9 @@ def get_list_of_window_slices(
 
     accumulated_list_of_windows = []
     for ifilename in file_names:
-
+        if ".tif.aux.xml" in ifilename:
+            continue
+        
```

Comment:
```suggestion

```
I prefer to remove this and be careful only to include tif file in the file_names (e.g. using this `glob("*.tif")`)